core/domain/feedback_services.py

```diff
@@ -83,42 +83,18 @@ def _create_models_for_thread_and_first_message(
     thread.entity_type = entity_type
     thread.entity_id = entity_id
     thread.original_author_id = original_author_id
-    # The feedback analytics jobs rely on the thread status being set to 'open'
-    # when a new thread is created. If this is changed, changes need to be made
-    # there as well.
-    thread.status = feedback_models.STATUS_CHOICES_OPEN
     thread.subject = subject
     thread.has_suggestion = has_suggestion
     thread.message_count = 0
+    # The feedback analytics jobs relies on new thread having an initial status
+    # of 'open' when a new thread is created. If this is changed, changes need
+    # to be made there as well.
+    thread.status = feedback_models.STATUS_CHOICES_OPEN
     thread.put()
-    create_message(
-        thread_id, original_author_id, feedback_models.STATUS_CHOICES_OPEN,
-        subject, text)
+    create_message(thread_id, original_author_id, thread.status, subject, text)
     return thread_id
 
 
-def create_thread(
```

Comment:
Is this function never used? If so, shouldn't some tests also be deleted?

Reply:
Ah, the diff just makes things look weird. I'm not deleting `create_thread`, I'm deleting the helper function `_create_models_for_thread_and_first_message `.